README.md

```diff
@@ -42,7 +42,7 @@ on [Filecoin Slack](https://filecoinproject.slack.com/)! 👋
 
 ### Node hardware requirements
 
-- Linux server with a static public IPv4 address
+- Linux server with a static public IPv4 address in a unique /24 CIDR block
```

Comment:
Will the typical node op understand "/24 CIDR block"?

Reply:
God I hope so, but easily google-able, also open to suggestions

Reply:
Can you elaborate on the uniqueness? Is it unique across the whole Saturn network? Or unique for a specific operator's wallet address?

Reply:
@DiegoRBaquero maybe best to just provide an example here to supplement the docs on the requirement.